tools/bake.py

```diff
@@ -463,12 +493,13 @@ def perform_bake(self, context):
         prioritize_face = context.scene.bake_prioritize_face
         prioritize_factor = context.scene.bake_face_scale
         uv_overlap_correction = context.scene.bake_uv_overlap_correction
-        margin = 0.01
+        margin = 2/resolution #4 pixels of margin no matter image size.
```

Comment:
Not sure if 4 pixels will be enough? Have you tested end results anywhere?

Reply:
I will increase to 8 since it's easily changable, I will do the same for the solid squares too